documentation/docs/pages/operators/nodes/nym-node/setup.mdx

```diff
@@ -0,0 +1,179 @@
+{/*
+import { MyTab } from '../../../../components/operators/nodes/node-run-command-tabs'
+import Mixnode from './snippets/mixnode-run-tab-snippet.mdx';
+import EntryGateway from './snippets/entry-gateway-run-tab-snippet.mdx';
+import ExitGateway from './snippets/exit-gateway-run-tab-snippet.mdx';
+import Mixnode from './snippets/mixnode-run-tab-snippet.mdx';
+*/}
+import { Callout } from 'nextra/components';
+import { Tabs } from 'nextra/components';
+import { RunTabs } from 'components/operators/nodes/node-run-command-tabs';
+import { VarInfo } from 'components/variable-info.tsx';
+import { MigrateTabs } from 'components/operators/nodes/node-migrate-command-tabs'
+
+# Nym Node Setup & Run
+
+This documentation page provides a guide on how to set up and run a [NYM NODE](nym-node.md), along with explanations of available flags, commands, and examples.
+
+## Current version
```

Comment:
Maybe we remove this for now

Reply:
I left all cmdrun for easier location post merge